tbtaf/databridge/TBTAFDataBridge.py

```diff
@@ -0,0 +1,178 @@
+from __future__ import absolute_import
+import cx_Oracle
+import os
+from common.suite import TBTestSuite
```

Comment:
Please use the same design approach that I mentioned for the TBTAFPublisher

Reply:
implemented